resources/helm/overlays/istio-control/istio-discovery/templates/validatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,141 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: istio-validator-{{ .Values.revision | default "default" }}-{{ .Release.Namespace }}
+  labels:
+    app: istiod
+    release: {{ .Release.Name }}
+    istio: istiod
+    istio.io/rev: {{ .Values.revision | default "default" | quote }}
+webhooks:
+  # Webhook handling per-revision validation. Mostly here so we can determine whether webhooks
+  # are rejecting invalid configs on a per-revision basis.
+  - name: rev.validation.istio.io
+    clientConfig:
+      # Should change from base but cannot for API compat
+      {{- if .Values.base.validationURL }}
+      url: {{ .Values.base.validationURL }}
+      {{- else }}
+      service:
+        name: istiod-{{ .Values.revision | default "default" }}
+        namespace: {{ .Release.Namespace }}
+        path: "/validate"
+      {{- end }}
+    namespaceSelector:
+      matchExpressions:
+      - key: maistra.io/member-of
+        operator: In
+        values:
+        - {{ .Release.Namespace }}
+    objectSelector:
+      matchExpressions:
+      - key: maistra-version
+        operator: DoesNotExist
+    rules:
+      - operations:
+        - CREATE
+        - UPDATE
+        apiGroups:
+        - authentication.maistra.io
+        apiVersions:
+        - "*"
+        resources:
+        - "*"
+      - operations:
+        - CREATE
+        - UPDATE
+        apiGroups:
+        - rbac.maistra.io
+        apiVersions:
+        - "*"
+        resources:
+        - "*"
+
+      - operations:
+          - CREATE
+          - UPDATE
+        apiGroups:
+          - security.istio.io
+        apiVersions:
+          - v1beta1
+          - v1
+        resources:
+          - authorizationpolicies
+          - requestauthentications
+
+      - operations:
+          - CREATE
+          - UPDATE
+        apiGroups:
+          - security.istio.io
+        apiVersions:
+          - v1beta1
+        resources:
+          - peerauthentications
+
+      - operations:
+          - CREATE
+          - UPDATE
+        apiGroups:
+          - networking.istio.io
+        apiVersions:
+          - v1alpha3
+          - v1beta1
+        resources:
+          - destinationrules
+          - gateways
+          - serviceentries
+          - sidecars
+          - virtualservices
+          - workloadentries
+          - workloadgroups
+
+      - operations:
+          - CREATE
+          - UPDATE
+        apiGroups:
+          - networking.istio.io
+        apiVersions:
+          - v1alpha3
+        resources:
+          - envoyfilters
+
+      - operations:
+          - CREATE
+          - UPDATE
+        apiGroups:
+          - networking.istio.io
+        apiVersions:
+          - v1beta1
+        resources:
+          - proxyconfigs
+
+      - operations:
+          - CREATE
+          - UPDATE
+        apiGroups:
+          - telemetry.istio.io
+        apiVersions:
+          - v1alpha1
```

Comment:
```suggestion
          - v1alpha1
          - v1
```

Reply:
The [CRD manifest](https://github.com/Maistra/istio-operator/blob/maistra-2.6/resources/helm/v2.6/istio-init/files/telemetries.telemetry.istio.io.crd.yaml#L37) contains only version v1alpha1. Maybe v1 was introduced in a later Istio version.